spec/models/article_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :lede }
+    it { is_expected.to have_db_column :body }
+    it { is_expected.to have_db_column :author }
+    it { is_expected.to have_db_column :date }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :lede } 
+    it { is_expected.to validate_presence_of :body }
+    it { is_expected.to validate_presence_of :author } 
+    it { is_expected.to validate_presence_of :date }
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:article)).to be_valid 
```

Comment:
Please include FactoryBot Matchers in your RSpec configuration (And Cucumber configuration) so that you can shorten this to 
```
expect(create(:article)).to be_valid 
```

Reply:
Done

Reply:
Partially... "And Cucumber configuration"

Reply:
Not sure where else you mean -- I added `World(FactoryBot::Syntax::Methods)` in `features/support/env.rb`, per the instructions [on this page](https://class.craftacademy.co/courses/course-v1:CraftAcademy+CA-CC-01+2018/courseware/cb5cc85a2e4a416a8d323d355fbcc799/fba6d0747a3d47a5b7cc4f802f8a997a/2?activate_block_id=block-v1%3ACraftAcademy%2BCA-CC-01%2B2018%2Btype%40vertical%2Bblock%40c56275b6744e4b0c957e7b3e7e84a06f).

Reply:
Okay, but you still use `FactoryBot.create` vs `create` in your step definitons.

Reply:
Ah, thanks for clarifying. Finally figured this one out.